assets/components/contribute/contribute.jsx

```diff
@@ -7,29 +7,44 @@ import React from 'react';
 import PageSection from 'components/pageSection/pageSection';
 import Secure from 'components/secure/secure';
 import type { Node } from 'react';
-
+import { classNameWithModifiers } from 'helpers/utilities';
 
 // ----- Types ----- //
 type PropTypes = {
   copy: string | Node,
   children: Node,
+  desktopAboveTheFoldVariant?: ?('control' | 'variant' | 'notintest'),
 };
 
 // ----- Component ----- //
 
 export default function Contribute(props: PropTypes) {
+  let modifiers = [];
+  let heading = 'Contribute';
+
+  if (props.desktopAboveTheFoldVariant) {
+    if (props.desktopAboveTheFoldVariant === 'variant') {
```

Comment:
`if (props.desktopAboveTheFoldVariant && props.desktopAboveTheFoldVariant === 'variant')` would be clearer I think